frontend/server/src/Controllers/Contest.php

```diff
@@ -2281,6 +2307,8 @@ public static function apiCreate(\OmegaUp\Request $r) {
             null :
             join(',', $r['languages'])
         );
+        $forTeams = $r->ensureOptionalBool('contest_for_teams') ?? false;
+        $teamsGroupsAlias = $r->ensureOptionalString('teams_group_alias');
```

Comment:
esto no tendría que ser no-opcional si es para equipos?

```suggestion
        $teamsGroupsAlias = $forTeams ? $r->ensureString('teams_group_alias', fn (string $alias) => \OmegaUp\Validators::alias($alias)) : null;
```

aunque no sea opcional, lo que sí es que hay que hacer es asegurarnos que sea un alias válido.

Reply:
Lo había dejado opcional, pensando en el caso de que aún no hayan creado el grupo y lo puedan agregar posteriormente cuando editen el concurso.  Quizás en ese caso sólo sea necesario agregar una nota que indique esto.

Pero vamos a dejarlo como dices y sólo habrá que indicar en el proceso los pasos para crear el concurso (Primero crear el grupo y después crear el concurso)

Reply:
sí, así no podemos tener un concurso en un estado inválido!